src/Tests/runTest.ts

```diff
@@ -45,25 +45,33 @@ import {runTests} from '@vscode/test-electron';
 import * as path from 'path';
 import {argv} from 'process';
 
+// NOTE: Fix the version to 1.64.0 to avoid some errors by the latest version
+// TODO: Remove the limitation of the version
+const codeVersionForGeneral: string = '1.64.0';
+
+// NOTE: Fix the version to 1.48.0 to avoid some nyc errors by the version > 1.48.0
+// TODO: Remove the limitation of the version
+const codeVersionForCoverage: string = '1.48.0';
+
 async function main() {
-  let ci: boolean = argv.length > 2 && argv.slice(2).includes('ci');
+  const args: string[] = (argv.length > 2 && argv.slice(2)) || new Array<string>();
+  const isCoverage: boolean = args.includes('coverage') ? true : false;
+  const codeVersion: string = isCoverage ? codeVersionForCoverage : codeVersionForGeneral;
+  const isCiTest: boolean = args.includes('ci') ? true : false;
 
   try {
     const extensionDevelopmentPath = path.resolve(__dirname, '../../');
-    const extensionTestsPath = (ci) ?
-        path.resolve(extensionDevelopmentPath, 'out', 'Tests', 'indexCI') :
-        path.resolve(extensionDevelopmentPath, 'out', 'Tests', 'index');
+    const extensionTestsPath = path.resolve(extensionDevelopmentPath, 'out', 'Tests', 'index');
     const testWorkspace = path.resolve(extensionDevelopmentPath);
-    // NOTE: Fix the version to 1.64.0 to avoid some errors by the latest version
-    // TODO: Remove the limitation of the version
     await runTests({
-      version: '1.64.0',
+      version: codeVersion,
       extensionDevelopmentPath,
       extensionTestsPath: extensionTestsPath,
-      launchArgs: [testWorkspace]
+      launchArgs: [testWorkspace],
+      extensionTestsEnv: {isCoverage: String(isCoverage), isCiTest: String(isCiTest)}
```

Comment:
Maybe?

```suggestion
      extensionTestsEnv: {isCoverage: string(isCoverage), isCiTest: string(isCiTest)}
```

Reply:
I misunderstand. the `String` is a function like `toString`.